django/contrib/admin/checks.py

```diff
@@ -20,6 +20,15 @@
 from django.utils.inspect import get_func_args
 
 
+def issubclass_skip_raise_exception(cls, classinfo):
+    """Handles TypeError in case `cls` is not a class
```

Comment:
I still don't like the `issubclass_skip_raise_exception` name. It just feels cumbersome. I suggest:
```
def _issubclass(cls, classinfo)
    """
    issubclass() variant that doesn't raise an exception if cls isn't a
    class.
    """
```
I think the purpose is clear enough.

Reply:
Okay, sounds good to me.